### PR TITLE
[relates #1041] Rest smoke test

### PR DESCRIPTION
**Pull request by Stefan365**
relates FUSEQE-1041

//test: @smoke

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java [hunk 1]
```diff
@@ -121,12 +123,11 @@ public void insertsIntoTable(String tableName, DataTable data) {
                     sql = String.format(sql, next) + ")";
                 }
             }
+            log.info("SQL query: *{}*", sql);
```
Comment (tplevko) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java]: Please lower the log to debug, so the logs would be more clear

File: rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/IntermediateSteps.java [hunk 2]
```diff
@@ -56,4 +68,9 @@ public void addLogStep() {
         steps.getStepDefinitions().add(new StepDefinition(basicFilter));
     }
 
+    @And("^create basic DB to DB filter step for \"([^\"]*)\" with word \"([^\"]*)\"$")
```
[the remark this answers is not in the document]
Comment (Stefan365) [reply rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/IntermediateSteps.java]: no, that's mistake.

**Comment by Stefan365** [comment]
comments impleneted & pushed.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
add jenkins job. Test fails because PostgresDB is not created after "clean application state"

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**